server/container_create_linux.go

```diff
@@ -820,6 +827,12 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 		makeOCIConfigurationRootless(specgen)
 	}
 
+	if cdiDevices != nil {
+		if err := injectCDIDevices(ctx, specgen, cdiDevices); err != nil {
```

Comment:
Can we omit the `nil` check and let `injectCDIDevices` do nothing if an empty slice is passed in?

Reply:
Update the implementation like that.